src/node_binding.cc

```diff
@@ -463,6 +463,10 @@ void DLOpen(const FunctionCallbackInfo<Value>& args) {
     }
 
     if (mp != nullptr) {
+      if (mp->nm_context_register_func == nullptr) {
+        if (env->force_context_aware())
+          env->ThrowError("Loading non context-aware native modules has been disabled in this process.");
```

Comment:
I think the linter would fail on this because it exceeds 80 characters.

Ideally, it would also be good for new errors to be listed in `node_errors.h` (and `doc/api/errors.md`) with an error code and a pre-defined error message.